packages/react-components/src/components/FileCard.tsx

```diff
@@ -44,15 +46,42 @@ export interface _FileCardProps {
   actionHandler?: () => void;
 }
 
+/**
+ * Strings of {@link _FileCard} that can be overridden.
+ *
+ * @public
+ */
+export interface FileCardStrings {
+  /** Aria label to notify user about the progress of file upload. */
+  uploadProgress?: string;
```

Comment:
Not able to imagine what this string would look like?
How is it different from `uploading` string?
Could you add some examples to the doc block?

Reply:
removed this string, was used to indicate the file progress with file name but as the file name is already announced , we don't need this string now